proteus/mprans/SW2DCV.h

```diff
@@ -2067,9 +2056,9 @@ namespace proteus
 	  ij=0;
 	  for (int i=0; i<numDOFsPerEqn; i++)
 	    {
-	      double hi = h_dof_lstage[i]; // solution at time tn for the ith DOF
-	      double hui = hu_dof_lstage[i]; 
-	      double hvi = hv_dof_lstage[i]; 
+	      double hi = h_dof_old[i]; // solution at time tn for the ith DOF
+	      double hui = hu_dof_old[i]; 
+	      double hvi = hv_dof_old[i]; 
```

Comment:
Similar to above, is this a permanent change or hack for the RK stages?

Reply:
This is permanent. The vectors h_dof_old, etc are updated after every stage of the SSP method.